etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/sftp/SFTP.java

```diff
@@ -176,11 +178,13 @@ private void transferFile(Session session, String remoteFile, String localFile)
             } catch (Exception e) {
                 LOGGER.error(e);
                 try {
-                    if (fos != null) fos.close();
+                    if (fos != null)
+                        fos.close();
                 } catch (Exception ee) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)